docs/en/binance-loom-ethereum.md

```diff
@@ -0,0 +1,512 @@
+    const tokenAddress = Address.fromString(this.extdevNetworkConfig['chainId'] + ':' + loomCoinContractAddress)
+    const mainNetContractAddress = rinkebyBEP2Token.networks[this.rinkebyNetworkConfig['networkId']].address
+    const gatewayContract = this.extdev2RinkebyGatewayContract
+
+    const receiveSignedWithdrawalEvent = new Promise((resolve, reject) => {
+      let timer = setTimeout(
+        () => reject(new Error('Timeout while waiting for withdrawal to be signed')),
+        timeout
+      )
+      const listener = event => {
+        const tokenEthAddress = Address.fromString('eth:' + mainNetContractAddress)
+        if (
+          event.tokenContract.toString() === tokenEthAddress.toString() &&
+          event.tokenOwner.toString() === ownerMainnetAddr.toString()
+        ) {
+          clearTimeout(timer)
+          timer = null
+          gatewayContract.removeAllListeners(Contracts.TransferGateway.EVENT_TOKEN_WITHDRAWAL)
+          console.log('Oracle signed tx ', CryptoUtils.bytesToHexAddr(event.sig))
+          resolve(event)
+        }
+      }
+      gatewayContract.on(Contracts.TransferGateway.EVENT_TOKEN_WITHDRAWAL, listener)
+    })
+    await gatewayContract.withdrawERC20Async(
+      new BN(amountInt, 10),
+      tokenAddress,
+      ownerMainnetAddr
+    )
+    console.log('before receiveSignedWithdrawalEvent')
+    await receiveSignedWithdrawalEvent
+  }
+```
+
+One thing to note. As seen above, calling `await gatewayContract.withdrawERC20Async` creates a pending withdrawal. The pending withdrawal is picked up by the Gateway Oracle, which signs the withdrawal, and notifies the  Gateway. The Gateway emits an event to let the user know they can withdraw their token from the Ethereum Gateway to their Ethereum account by providing the signed withdrawal record.
+
+- Next, we fetch the signed withdrawal record with:
+
+```js
+async _getWithdrawalReceipt () {
+  const userLocalAddr = Address.fromString(this.accountMapping.plasma.toString())
+  const gatewayContract = this.extdev2RinkebyGatewayContract
+  const data = await gatewayContract.withdrawalReceiptAsync(userLocalAddr)
+  if (!data) {
+    return null
+  }
+  const signature = CryptoUtils.bytesToHexAddr(data.oracleSignature)
+  return {
+    signature: signature,
+    amount: data.value.toString(10),
+    tokenContract: data.tokenContract.local.toString()
+  }
+}
+```
+
+- lastly, let's withdraw the tokens to our Rinkeby account by calling the `withdrawCoinsFromRinkebyGateway` function and passing it the `data` structure containing the signature:
+
+
+```js
+async _withdrawCoinsFromRinkebyGateway (data) {
+  const rinkebyContractAddress = rinkebyBEP2Token.networks[this.rinkebyNetworkConfig['networkId']].address
+  const userRinkebyAddress = this.accountMapping.ethereum.local.toString()
+  const tx = await this.rinkeby2ExtdevGatewayContract.methods
+    .withdrawERC20(data.amount.toString(), data.signature, rinkebyContractAddress)
```

Comment:
This is the API for the old gateway, won't work with the current extdev gateway.